### PR TITLE
Forensics QOL Changes

### PR DESCRIPTION
**Pull request by Crosarius**
Makes it so that using the forensics item on certain machinery such as airlocks, alarms, APCs, buttons, windoors etc doesn't apply fingerprints/fibers. This is necessary because these machines will call add_fingerprint BEFORE the forensic kit is used, thus polluting the sample with your own fibers, which is extremely annoying

Also made it so that using a forensics item on a fire alarm or button doesn't make you trigger them, because that's also really annoying.

This is take 2 because the first time I messed up my branches/forks because my brain is smooth.

### CODE REVIEW

**Comment by Crosarius** [comment]
!review

**Comment by Crosarius** [comment]
I genuinely don't understand what happened there. When I looked at the merge conflict there were no changes in the master - it was just the same code without my code. I had to go through and just delete the master version and leave the forensics version behind because it marked the entirety of the file as a conflict.

File: code/game/machinery/Sleeper.dm [hunk 1]
```diff
@@ -158,7 +158,8 @@
 	return attack_hand(user)
 
 /obj/machinery/sleeper/attackby(var/obj/item/I, var/mob/user)
-	add_fingerprint(user)
+	if(!istype(I, /obj/item/forensics))
```
Comment (NonQueueingMatt) [inline code/game/machinery/Sleeper.dm]: Considering how much this is used, this should probably be an atom flag. Or just put the check in add_fingerprint tbh.

**Comment by Crosarius** [comment]
This still has changes requested but I can't clear the tag. I fixed the conflicts.

**Pull request merged**